hexrd/instrument.py

```diff
@@ -1298,13 +1298,6 @@ def simulate_powder_pattern(self,
                             self.simulated_spectrum.x,
                             self.simulated_spectrum.y + self.background.y)
 
-            # normalize everything to 0-1
-            mi = img.min()
-            ma = img.max()
-
-            if(ma > mi):
-                img = (img - mi) / (ma - mi)
-
```

Comment:
We should make sure this won't have any unexpected side effects.

Reply:
@saransh13 -- I asked you about this and it seemed ok, but let's verify?